account-kit/react/src/icons/illustrations/utils.ts

```diff
@@ -0,0 +1 @@
+export type IllustrationStyle = "outline" | "linear" | "filled" | "flat";
```

Comment:
there may be a better place for this depending on whether we move forward with CSS variables or decide to set a config value, but it's here for now